src/components/comment/ActionDropdown.scss

```diff
@@ -0,0 +1,3 @@
+.bdl-ThreadedComment-actionDropdown .icon-ellipsis {
```

Comment:
It looks like the intention is for this component to not be aware of its children, but this style breaks that encapsulation.

Reply:
I've refactored the `ActionDropdown` and it's treated as part of the `Comment` component broken up from it. And the button with ellipsis icon is a static part of it that needed a slight design change.